kind-setup/prometheus/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM prom/prometheus
+ADD prometheus.yml /etc/prometheus/
```

Comment:
There's no need to build a separate Docker image for Prometheus. We can simply specify a volume mount in the `Deployment.Spec.Template.Spec` (see below)

Reply:
Removed the Dockerfile.